fuzzinator/mongo_driver.py

```diff
@@ -57,6 +58,8 @@ def all_issues(self):
         return list(self._db.fuzzinator_issues.find({}))
 
     def find_issue_by_id(self, id):
+        if not isinstance(id, ObjectId):
```

Comment:
This condition might not be needed at all. `id = ObjectId(id)` might also work if `id` is an instance of `ObjectId`. (see: https://github.com/mongodb/mongo-python-driver/blob/master/bson/objectid.py#L204) Simply writing `id = ObjectId(id)` might better express the intent that we want to _enforce_ that the `id` is an `ObjectId`, unconditionally. And leave the corner cases of creation to the implementation of `ObjectId`.

Same comment applies to `remove_issue_by_id`.

Reply:
Done.